data-service/pom.xml

```diff
@@ -26,11 +26,13 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
+    <!--  *********** Aritfact ID ***********    -->
     <artifactId>data-service</artifactId>
+    <url>https://github.com/gchq/Palisade-services/tree/develop/data-service</url>
+    <!--  *********** Aritfact ID ***********    -->
 
     <properties>
         <eureka.version>1.9.16</eureka.version>
-        <databind.version>2.9.8</databind.version>
```

Comment:
Why has this gone?

Reply:
There was a duplication of the Jackson Databind license information. This value was the older version and was not being used within the pom so I removed it.